### ISSUE TITLE
phero-server errored with message: spawn npm ENOENT (Windows)

### ISSUE THREAD

**Issue by draylegend** [phero-hq/phero#99]
> phero/api/

`npm init -y`
`npm install typescript --save-dev`
`npx tsc --init`
`npx phero init server`
> Ready to go! Run `npx phero` again to continue.

`npx phero`
```js
phero-server @ ./ ◢ Loading...
C:\Users\adm1n\AppData\Local\npm-cache\_npx\9d0fc90ec0bbaf30\node_modules\yoga-layout-prebuilt\yoga-layout\build\Release\nbind.js:53
        throw ex;
        ^

Error: phero-server is not installed in D:\dev\web\tests\phero\api
    at ChildProcess.<anonymous> (C:\Users\adm1n\AppData\Local\npm-cache\_npx\9d0fc90ec0bbaf30\node_modules\phero\dist\process.js:37:27)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess._handle.onexit (node:internal/child_process:289:12)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v18.12.1
```

### Expected

server should start

**Comment by Jpunt** [comment]
@draylegend can you please test this with the upcoming version? I’ve released a fix for that yesterday (#94)

```
npx phero@next
```

**Comment by draylegend** [comment]
same error

```js
npx phero@next
◤
phero-server @ . ◢ Loading...
C:\Users\adm1n\AppData\Local\npm-cache\_npx\deae763e71cc8ce9\node_modules\yoga-layout-prebuilt\yoga-layout\build\Release\nbind.js:53
        throw ex;
        ^

Error: phero-server errored with message: spawn npm ENOENT
    at ChildProcess.<anonymous> (C:\Users\adm1n\AppData\Local\npm-cache\_npx\deae763e71cc8ce9\node_modules\phero\dist\process.js:36:15)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess._handle.onexit (node:internal/child_process:289:12)
    at onErrorNT (node:internal/child_process:476:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v18.12.1
```

### package.json

```
typescript: 4.9.4
@phero/server: 0.9.14-alpha.0
```

The command `npx phero@next` init server didn't install the `next` version (0.9.4-alpha.0). Had to update it manually in `package.json`. But after that, the server didn't start.

**Comment by Jpunt** [comment]
@draylegend alright, I'll go get a Windows machine up and running to properly debug this. Will get back to you when I know more!

**Comment by draylegend** [comment]
It also didn't work in a `Windows sandbox`

**Comment by draylegend** [comment]
> When you install the cli-package in your project, it would make sense `npx` would take that one instead of fetching it remotely. I'd recommend not installing it locally, but always use `npx` instead. This is different from the `@phero/client` and `@phero/server` packages though, maybe we should be explaining this better in the readme.

I followed the instructions in `Getting Started`:

```
# Go to your backend project directory:
cd ./api
​
# Make sure npm and TypeScript are ready to go:
npm init -y
npm install typescript --save-dev
npx tsc --init
​
# Add Phero to your backend project:
npx phero init server
```

then adjusted `tsconfig.json`

```
{
  "compilerOptions": {
    ...
    "rootDir": "src",
    "outDir": "dist"
  }
}
```

I had to manually update to the `next` version, because `npx phero init server` didn't do it.

**Comment by Jpunt** [comment]
Ah right, in that case I misunderstood. Will take a closer look 👍

**Comment by Jpunt** [comment]
@draylegend this should be fixed in `phero@0.9.14-alpha.1`, which with I've successfully ran `npx phero@next` on Windows. Feel free to reach out when you hit anything.

**Comment by Jpunt** [comment]
@draylegend 0.9.14 is released 🙂
